config/locales/en.yml

```
@@ -11,6 +11,15 @@ en:
       scitt: SCITT
       university: University
       lead_school: Lead partner
+    school_options_form_component:
+      add_organisation: Add Organisation
```

Everywhere else, we are using the less formal 'Add organisation'. 
```suggestion
      add_organisation: Add organisation
```